microconfig/utils/src/test/java/io/microconfig/utils/StringUtilsTest.java

```diff
@@ -1,12 +1,33 @@
 package io.microconfig.utils;
 
 import org.junit.jupiter.api.Test;
+import java.util.ArrayList;
 
 import static io.microconfig.utils.StringUtils.like;
 import static io.microconfig.utils.StringUtils.toLowerHyphen;
 import static org.junit.jupiter.api.Assertions.*;
 
 class StringUtilsTest {
+
+    @Test
+    void testIsEmpty() {
+        assertTrue(StringUtils.isEmpty(""));
+        assertTrue(StringUtils.isEmpty(null));
+
+        assertFalse(StringUtils.isEmpty("foo"));
+    }
+
+    @Test
+    void testUnixLikePath() {
+        assertEquals("B/ar", StringUtils.unixLikePath("B\\ar"));
+    }
+
+    @Test
+    void testReplaceMultipleSpaces() {
+        assertEquals("B a r",
+                StringUtils.replaceMultipleSpaces("  B a r  "));
```

Comment:
In this case, I would `import static io.microconfig.utils.StringUtils.*` and call the methods unqualified, since that's what the original tests do.

Reply:
Done.. I replaced everything other than `isEmpty()` as this is quite a common method